app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -264,8 +264,16 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
     const List<String> _failedStates = <String>['FAILED', 'ABORTED'];
     const List<String> _succeededStates = <String>['COMPLETED', 'SKIPPED'];
     final GraphQLClient cirrusClient = await config.createCirrusGraphQLClient();
-    final List<Map<String, dynamic>> cirrusStatuses =
+    final List<CirrusResult> cirrusResults =
         await queryCirrusGraphQL(sha, cirrusClient, log, name);
+    if (!cirrusResults
+        .any((CirrusResult cirrusResult) => cirrusResult.branch == 'master')) {
```

Comment:
Should we use a branch passed as a parameter instead of fixing it to master?

Reply:
Done.